go-apps/meep-gis-engine/server/gis-automation.go

```diff
@@ -0,0 +1,491 @@
+/*
```

Comment:
General comment: 
REST calls to the sandbox manager could have been sent using the message bus

Reply:
Agreed. Note that this will require some Sandbox Controller refactoring to clearly separate REST endpoint functionality from Event handling functionality. For now the added code uses the same method as the initial MOBILITY event.